spec/winnow/model_spec.rb

```diff
@@ -159,7 +159,7 @@
           [Struct.new(:columns, :type).new(["name"], :fulltext)]
         end
 
-        expect_any_instance_of(ActiveRecord::Relation).to receive(:where).with(fts_scope, "test*", "%test%")
+        expect_any_instance_of(ActiveRecord::Relation).to receive(:where).with(fts_scope, "", "%test%")
```

Comment:
Not sure if the blank here is gonna be okay ie does it throw some huge inefficiency into the query?